test/clt-tests/installation/rhel-dev-update.rec

```diff
@@ -6,7 +6,7 @@
 ––– input –––
 yum -y --enablerepo manticore-dev update manticore manticore-extra manticore-common manticore-server manticore-server-core manticore-tools manticore-executor manticore-buddy manticore-backup manticore-columnar-lib manticore-server-core-debuginfo manticore-tools-debuginfo manticore-columnar-lib-debuginfo  manticore-icudata manticore-galera > /dev/null 2>&1; echo $?
 ––– output –––
-0
+1
```

Comment:
Instead of this, let's remove the previous packages according to the instruction on https://manticoresearch.com/install :
```
sudo yum --setopt=tsflags=noscripts remove 'manticore*'
```

Reply:
Done